src/demark_world/iopaint/cli.py

```
@@ -9,9 +9,9 @@
 from typer import Option
 from typer_config import use_json_config
 
-from src.demark_world.iopaint.const import *
-from src.demark_world.iopaint.runtime import check_device, dump_environment_info, setup_model_dir
-from src.demark_world.iopaint.schema import (
+from demark_world.iopaint.const import *
```

Ruff (0.15.14)

[error] `from demark_world.iopaint.const import *` used; unable to detect undefined names

(F403)